### PR TITLE
Fix Pip and Pipenv version to 18 and 2018.7.1

### PR DESCRIPTION
**Pull request by Mallear**
Recently, Pip 18.1 and Pipenv 2018.7.1/2018.10.9 conflict a lot during dependancies resolution. Setting version to 18.0 and 2018.7.1 fix it.

I have also added some packages needed to make cpython libs available and usable in dind-aws image.

### CODE REVIEW

File: aws/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,8 @@ ARG CI_HELPER_VERSION
 WORKDIR /root
 RUN echo "Install AWS" && \
     apk add -q --no-cache bash ca-certificates curl git make openssh-client python py-pip rsync zip && \
-    pip -q install awscli boto3 PyYAML && \
+    pip -q install pip==18 \
```
Comment (mremi) [inline aws/Dockerfile]: Should add arguments to configure it in `.travis.yml`

Comment (Mallear) [reply aws/Dockerfile]: Done

**Comment by Mallear** [comment]
I have fixed Dockerfile templates to allow installing a specific version of pip

**Comment by Mallear** [comment]
@mremi It finally works !

**Comment by mremi** [comment]
Please rebase

**Pull request merged**